.eslintignore

```diff
@@ -2,4 +2,5 @@
 /cjs
 /docs
 /esm
-/tmp
\ No newline at end of file
+/tmp
+/tests/nodenext/index.ts
```

Comment:
Why is this needed here? We don't have entries for the other test files.

Reply:
I was receiving a lint error in that file. I have changed the configuration of .eslintrc.json to include the tsconfig file in that folder. It seems to work.

Reply:
How can I fix that issue? My intellisense is getting the correct typings for papr.

Reply:
Adding a top level `/* eslint-disable @typescript-eslint/no-unsafe-call */` in that file should be fine.

Reply:
done! Thanks!